contracts/examples/esdt-nft-marketplace/README.md

```diff
@@ -0,0 +1,104 @@
+
+As we all know, there really isn't such thing as "unique data" when it comes to computers. They copy and share data all the time. The key difference here is the immutability of data on the blockchain: Once an NFT is created, you will be able to trace its history through time, without having to worry about a third party modifying, or even deleting that said NFT.  
+
+But what if you decide you don't really have any value for said collectible item? That's where the NFT marketplace comes in.  
+
+## Difference between ESDT NFTs and ERC1155 NFTs
+
+Before we start with the marketplace, there's something else that needs to be covered. If you're familiar with Ethereum, you probably heard about the ERC1155 standard for NFTs. There are a few problems with that standard.  
+
+The main problem is that it is very centralized. A single smart contract holds the data for all NFTs, and all the related operations, like transfers, are also done through that contract. This can lead to trust issues or bottlenecks.  
+
+Another problem is that using said NFTs as payment for smart contracts is very difficult. It requires a call to the main ERC1155 contract, which then calls the receiver smart contract, which has to implement a very specific method for receiving the token. As you can probably guess, this adds a lot of overhead and costs a lot of gas. And now the receiver smart contract owner has another problem: how to get that NFT to its own user account? This requires yet another call to the ERC1155 contract, through the receiver contract, which is another two SC calls. More overhead, more gas.  
+
+By implementing NFTs on the protocol level, we've solved those problems by treating NFTs as ESDTs with additional metadata. This means your user account can *own* the NFT natively instead of relying on an external contract. You can also transfer that NFT at will, and even use it as payment for smart contracts. And what's more, said contract doesn't even need to implement some standardized interface!  
+
+An ESDT NFT has a token identifier (like all the rest), which is also referred to as its "type" or "class". Then, a type can have "subtypes" or "subclasses", differentiated by `nonce`, `name`, and additional user-defined `attributes`.  
+
+Now that we've hyped ESDT NFTs, it's time to see how you can sell them through the marketplace.  
+
+## The Marketplace
+
+The marketplace has been designed to be as simple as possible. All it does is accept an NFT and auction it with your terms. It accepts bids from other accounts, and concludes the auction by either sending the token back to you if no bids were received, or send you the respective bid amount and the token to the new owner.
```

Comment:
I don't think "concludes"  can be used like that, use "finishes" instead

better: ... and finishes the auction by either sending the token to the winner, or back to the owner if no bids were received.